generator/python/resources/templates/criteo_rest.mustache

```diff
@@ -0,0 +1,111 @@
+import json
+from datetime import datetime, timedelta
+from {{packageName}}.rest import RESTClientObject
+from {{packageName}}.exceptions import ApiException
+from {{packageName}}.rest import RESTClientObject
+
+class CriteoRESTClientObject(RESTClientObject):
+    def __init__(self, configuration, pools_size=4, maxsize=None):
+        super().__init__(configuration, pools_size, maxsize)
+
+        self.token = None
+        self.client_id = configuration.username
+        self.client_secret = configuration.password
+        self.grant_type = 'client_credentials'
+        self.access_token = configuration.access_token
+        self.host = configuration.host

```

Comment:
This seems duplicated from the RESTClientObject constructor. Is it useful?

Reply:
I have removed our customized rest.mustache so that it is now the generator embedded rest.mustache is used. The corresponding rest.py doesn't have this code, that was actually our customization of the overriding rest.mustache